website/static/js/registrationUtils.js

```diff
@@ -198,6 +204,18 @@ var Question = function(data, id) {
     });
 
     /**
+     * @returns {Boolean} true if the value <input> does validate
+     **/
+    self.hasErrors = ko.computed(function() {
+        if( self.required ){
```

Comment:
@barbour-em Are you seeing this exception or expecting it to get raised?

I'm not sure why it would get raised. ```self.value.isValid``` by all appearances is a function.